src/main/java/com/github/caldav4j/methods/HttpCalDAVReportMethod.java

```diff
@@ -1,8 +1,11 @@
 package com.github.caldav4j.methods;
 
-import net.fortuna.ical4j.data.CalendarBuilder;
-import net.fortuna.ical4j.data.ParserException;
-import net.fortuna.ical4j.model.Calendar;
+import java.io.IOException;
+import java.io.InputStream;
```

Comment:
If that's just reordering imports, it would be great if you propose it in a separate PR.

If that's done via a tool, please share your configuration so that we can add it to the build pipeline.

Reply:
The re-ordering occurred when I deleted the logger declaration (line 37) and it was a automatic reorder made by eclipse; here's my config:
```
#Organize Import Order
3=com
2=org
1=javax
0=java
```